app/models/i14y_post_processor.rb

```diff
@@ -14,17 +14,12 @@ def post_process_results
     strip_highlighting unless @enable_highlighting
   end
 
-  def normalized_results
-    @results.map do |result|
-      {
-        title: result['title'],
-        url: result['link'],
-        description: result['body'],
-        updatedDate: parse_result_date(result['changed']),
-        publishedDate: parse_result_date(result['published_at']),
-        thumbnailUrl: result['thumbnail_url'] || nil
-      }
-    end
+  def normalized_results(total_results)
+    {
+      totalPages: total_pages(total_results),
```

Comment:
The `#total_pages` method is defined in the new `ResultsPostProcessor` base class.